### PR TITLE
Task-42410: Improve logs and implement malwareDetection jcr clean

### PR DESCRIPTION
**Pull request by azayati**
This PR will improve the error log message format on getInfectedItems method of MalwareDetectionJcrConnector and implement malware detection jcr connector clean

### CODE REVIEW

File: core/connector/src/main/java/org/exoplatform/ecm/connector/malwareDetection/MalwareDetectionJcrConnector.java [hunk 1]
```diff
@@ -44,53 +48,90 @@ public MalwareDetectionJcrConnector(InitParams initParams, RepositoryService rep
...
-                    infectedJcNode.getName(),
-                    infectedJcNode.getProperty(NodetypeConstant.EXO_LAST_MODIFIER).getString(),
+                    ((NodeImpl) infectedJcrNode).getIdentifier(),
+                    infectedJcrNode.getName(),
+                    infectedJcrNode.getProperty(NodetypeConstant.EXO_LAST_MODIFIER).getString(),
                     endTime - startTime);
       }
+    } catch (ItemNotFoundException e) {
+      long endTime = System.currentTimeMillis();
+      LOGGER.warn("service={} operation={} parameters=\"infectedJcrNodePath:{}\" status=ko duration_ms={} error_msg=The jcr node may be already cleaned by a previous malware detection operation:{}",
```
Comment (rdenarie) [inline core/connector/src/main/java/org/exoplatform/ecm/connector/malwareDetection/MalwareDetectionJcrConnector.java]:
```suggestion
      LOGGER.warn("service={} operation={} parameters=\"infectedJcrNodePath:{}\" status=ko duration_ms={} error_msg=\"The jcr node may be already cleaned by a previous malware detection operation:{}\"",
```

File: core/connector/src/main/java/org/exoplatform/ecm/connector/malwareDetection/MalwareDetectionJcrConnector.java [hunk 2]
```diff
@@ -44,53 +48,90 @@ public MalwareDetectionJcrConnector(InitParams initParams, RepositoryService rep
...
+                   MALWARE_DETECTION_FEATURE,
+                   MALWARE_INFECTED_JCR_NODE_DETECTION,
+                   infectedJcrNodePath,
+                   endTime - startTime);
     } catch (Exception e) {
-      LOGGER.error("Error when trying to get the infected jcr node informations from jcrNodeIdentifier={}", infectedJcrNodeIdentifier, e);
+      long endTime = System.currentTimeMillis();
+      LOGGER.error("service={} operation={} parameters=\"infectedJcrNodePath:{}\" status=ko duration_ms={} error_msg=Error when trying to get the infected jcr node information:{}",
```
Comment (rdenarie) [inline core/connector/src/main/java/org/exoplatform/ecm/connector/malwareDetection/MalwareDetectionJcrConnector.java]:
```suggestion
      LOGGER.error("service={} operation={} parameters=\"infectedJcrNodePath:{}\" status=ko duration_ms={} error_msg=\"Error when trying to get the infected jcr node information:{}\"",
```

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: core/connector/src/main/java/org/exoplatform/ecm/connector/malwareDetection/MalwareDetectionJcrConnector.java (hunk 1 above)
Comment (azayati) [reply core/connector/src/main/java/org/exoplatform/ecm/connector/malwareDetection/MalwareDetectionJcrConnector.java]: updated

File: core/connector/src/main/java/org/exoplatform/ecm/connector/malwareDetection/MalwareDetectionJcrConnector.java (hunk 2 above)
Comment (azayati) [reply core/connector/src/main/java/org/exoplatform/ecm/connector/malwareDetection/MalwareDetectionJcrConnector.java]: updated

**Review by rdenarie — APPROVED** [review]

**Pull request merged**